ansible/scripts/ansible.cfg

```diff
@@ -0,0 +1,2 @@
+[defaults]
+roles_path = ../roles
```

Comment:
From [ansible] (http://docs.ansible.com/ansible/intro_configuration.html)
"Changes can be made and used in a configuration file which will be processed in the following order:"
```
* ANSIBLE_CONFIG (an environment variable)
* ansible.cfg (in the current directory)
* .ansible.cfg (in the home directory)
* /etc/ansible/ansible.cfg
```

I'm not certain what current directory is defined as. I'd guess it would be the `scripts/` directory in our case